rts/lra/lra-coordinator-jar/pom.xml

```diff
@@ -12,35 +14,26 @@
     <packaging>jar</packaging>
 
     <properties>
-        <narayana.version>${project.version}</narayana.version>
+        <ipv6.server.jvm.args>
+            -Djboss.bind.address=[::1] -Djboss.bind.address.management=[::1] -Djboss.bind.address.unsecure=[::1]
```

Comment:
why do we need jboss specific sys props when this will be used in Quarkus based coordinator? Is it best to keep them in the shared module?

Reply:
We test the coordinator jar against a WildFly container.

Reply:
I think we should resolve this part of the conversation in order to keep the thread below in line

Reply:
@xstefank hopefully you can re-open if you disagree it is not directly related to the discussion below

Reply:
ok. I was just curious if JBoss specific properties will be propagated to the quarkus LRA coordinator. But if it's only for tests it should be ok.